docs/backends/TES.md

```diff
@@ -61,16 +61,29 @@ This backend supports CPU, memory and disk size configuration through the use of
     * Type: String (ex: "4 GB" or "4096 MB")
 * `disk` defines the amount of disk to use.
     * Type: String (ex: "1 GB" or "1024 MB")
+* `disks` accepts a GCP-style disk declaration and attempts to translate it for use on TES
+    * See table below for supported translations 
 * `preemptible` defines whether or not to use preemptible VMs.
     * Type: Boolean (ex: "true" or "false")
     * Integers are accepted and will be converted to boolean (true if > 0)
 
 If they are not set, the TES backend may use default values.
 
-**Azure**
+#### GCP `disks` to TES `disk` compatibility
+
+| GCP `disks` value                     | Supported | TES translation | Remark                            |
+|---------------------------------------|-----------|-----------------|-----------------------------------|
+| `local-disk 25 HDD`                   | ✅        | 25 GB disk      |                                   |
+| `local-disk 25 SSD`                   | ✅        | 25 GB disk      | Disk type info is dropped         |
+| `/some/mnt 25 SSD`                    | ❌        |                 | Custom mount points not supported | 
+| `local-disk 25 HDD, /some/mnt 50 SSD` | ❌        |                 | Multiple disks are not supported  | 
+            
+Note: if both `disk` and `disks` attributes are specified, the TES backend will automatically use the value in `disk` and not attempt to translate `disks`.
```

Comment:
The "Recognized Runtime attributes and Backends" table on [this page](https://cromwell.readthedocs.io/en/stable/RuntimeAttributes/) claims that only GCP supports `disks`. Meanwhile the detail section on `disks` on that same page suggests AWS kind of supports it but "the Disk size and disk type will be ignored".

Since I've already got all of this in my head, I think I'll update this page a bit more since it is about to receive a lot of traffic from TES people wanting to use `disks.`